dashboard.el

```diff
@@ -180,6 +163,78 @@ If MESSAGEBUF is not nil then MSG is also written in message buffer."
     (beginning-of-line)
     (widget-forward 1)))
 
+(defun dashboard-generate-banner ()
```

Comment:
They don't treat the banner as a widget https://github.com/syl20bnr/spacemacs/blob/b7e51d70aa3fb81df2da6dc16d9652a002ba5e6b/core/core-spacemacs-buffer.el#L827 and neither will we. The warnings in this case IS a widget that displays some list but the banner is not.